tests/providers/amazon/aws/sensors/test_sqs.py

```diff
@@ -305,3 +305,31 @@ def test_poke_do_not_delete_message_on_received(self, mock_conn):
         )
         self.sensor.poke(self.mock_context)
         assert mock_conn.delete_message_batch.called is False
+
+    @mock_sqs
+    def test_poke_batch_messages(self):
```

Comment:
@eladkal as the`num_batches` feature is added directly to `SqsSensor`, I added this test.

However, still thinking if this one is enough. The other existing tests look rather complete to serve as non-regression proof, IMHO.

If you have any other idea about other tests, feel free to suggest.